### DISCUSSION TITLE
Banque pour les guildes

### DISCUSSION THREAD

**Discussion by MF4db** [Crownicles/Crownicles#830] [Nouvelles idées]
Créer une banque pour chaque guildes où chaque membres peut déposer son argent et la récupérer pour partager les bénéfices.

**Comment by Gabrielre14** [comment]
Moi je pense plutôt à une trésorerie de guilde...
C'est-à-dire que chaque membre de la guilde peut faire un don d'une certaine somme. Tous les dons sont regroupés dans la trésorerie (qu'on pourrait faire apparaître avec une commande comme !treasury). Cet argent sert au chef ou à l'aîné d'acheter des options (chères) pour la guilde, comme une place dans le refuge (le prix augmente après chaque achat de place, comme pour l'inventaire, on pourrait aussi faire en sorte que le refuge de base soit moins grand pour forcer les chefs à acheter), une place pour la guilde (très cher), réduire le temps entre les gd, **possibilité de changer le nom de la guilde en payant**...
Ainsi, on pourrait aussi créer une autre récompense de guilde : de l'argent pour la trésorerie.

Bien entendu, le chef ou l'aîné n'a pas le droit de s'approprier cet argent pour lui... il pourrait peut-être en prélever une partie chaque semaine.
